tx_test.go

```diff
@@ -433,6 +433,283 @@ func TestTx_DeleteBucket_NotFound(t *testing.T) {
 	}
 }
 
+func TestTx_MoveBucket(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		srcBucketName           string
+		subBucketName           string
+		subBucketKey            string
+		subBucketValue          string
+		dstBucketName           string
```

Comment:
In your test cases, all these fields have exactly the same set of values, then it doesn't make sense to define them. Note: this is a generic comment, please think about to simplify the test case design.
```suggestion
```

Reply:
oh i am replacing this test, one sec i should have deleted it